src/main/java/io/github/fusionflux/portalcubed/packet/clientbound/OpenPedestalButtonConfigPacket.java

```diff
@@ -0,0 +1,38 @@
+package io.github.fusionflux.portalcubed.packet.clientbound;
+
+import org.quiltmc.qsl.networking.api.PacketSender;
+
+import io.github.fusionflux.portalcubed.content.button.pedestal.PedestalButtonBlockEntity;
+import io.github.fusionflux.portalcubed.content.button.pedestal.PedestalButtonConfigScreen;
+import io.github.fusionflux.portalcubed.content.prop.HammerItem;
+import io.github.fusionflux.portalcubed.packet.ClientboundPacket;
+import io.github.fusionflux.portalcubed.packet.PortalCubedPackets;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.player.LocalPlayer;
+import net.minecraft.core.BlockPos;
+import net.minecraft.network.FriendlyByteBuf;
+import net.minecraft.network.protocol.common.custom.CustomPacketPayload;
+import net.minecraft.resources.ResourceLocation;
+
+public record OpenPedestalButtonConfigPacket(BlockPos pedestalButtonPos) implements ClientboundPacket {
+	public OpenPedestalButtonConfigPacket(FriendlyByteBuf buf) {
+		this(buf.readBlockPos());
+	}
+
+	@Override
+	public void write(FriendlyByteBuf buf) {
+		buf.writeBlockPos(pedestalButtonPos);
+	}
+
+	@Override
+	public ResourceLocation id() {
+		return PortalCubedPackets.OPEN_PEDESTAL_BUTTON_CONFIG;
+	}
+
+	@Override
+	public void handle(LocalPlayer player, PacketSender<CustomPacketPayload> responder) {
```

Comment:
missing ClientOnly annotation

Reply:
None of the clientbound packets do this, guess ill make all of them do it